### PR TITLE
add ETHEREUM_DEV_CLUSTER flag

### PR DESCRIPTION
**Pull request by rasom**
Allows usage of dev clusters in jenkins builds. 
 
status: ready

### CODE REVIEW

File: modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java [hunk 1]
```diff
@@ -160,7 +162,7 @@ private void doStartNode(final String defaultConfig) {
         }
 
         int dev = 0;
-        if (this.debug) {
+        if (this.devCluster) {
```
Comment (oskarth) [inline modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java:165]: Why this change? I would imagine debug => devCluster, not other way around

**Review by oskarth — APPROVED** [review]

File: modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java (hunk 1 above)
Comment (rasom) [reply modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java:165]: `BuildConfig.DEBUG` is more about development and prod mode of application, when this `devCluster` means that devCluster can be used even if app wasn't compiled for local development.
